### DISCUSSION TITLE
Would it make sense to publish Twind as a single JS file?

### DISCUSSION THREAD

**Discussion by rgwood** [tw-in-js/twind#328] [Ideas]
Twind is very cool but the existing distribution mechanisms (Skypack, NPM) aren't a great fit for my needs. I just want a single .js file that I can serve up alongside my HTML.

It would be nice if Twind offered this; could be as simple as a `twind.js` file that gets published with each GitHub release.

**Comment by lukejacksonn** [comment]
Does the UMD build suite your needs? https://unpkg.com/twind@0.16.17/twind.umd.js

**Reply by rgwood** [reply]
It might! I think I was turned off by the "for legacy browsers" description but on second thought that might work. Will give it a try, thanks.
